appengine/accessible/workspaceview.component.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Blockly Games: Accessible
+ *
+ * Copyright 2016 Google Inc.
+ * https://github.com/google/blockly-games
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Angular2 Component that details how a Blockly.Workspace is
+ * rendered in AccessibleBlockly. Modified to run music code.
```

Comment:
One option (which I think is what you suggested) is to have the runCode() function live outside the WorkspaceView but have the button still live here. That would require the developer to create some kind of runCode() function in the global scope. Is that what you were suggesting?